readinglist/resource.py

```diff
@@ -273,8 +285,112 @@ def _extract_sorting(self, queryparams):
                 direction = -1 if order == '-' else 1
                 sorting.append((field, direction))
 
+        if not modified_field_used:
+            # Add a sort by the ``modified_field`` in descending order
+            # useful for pagination
+            sorting.append((self.modified_field, -1))
```

Comment:
I don't agree with this here. In the docs we said it would be ``-stored_on`` (see #84)

If you need this, put it only if pagination is active